src/components/Viewer.tsx

```diff
@@ -93,6 +197,7 @@ export const Viewer: Component = ({}) => {
         onMouseUp={mouseUp}
         onMouseMove={mouseMove}
         onWheel={wheel}
+        {...((getSettingOption('pwa_mounted') === 'true' && expectsTouch) ? binders() : noPWABinders())}
       >
         <div class="contents pointer-events-none select-none">
           <$for each={ActiveFields}>{(field) => <Label position={field.id}>{field.label}</Label>}</$for>
```

Comment:
Image labels overlap on small screens

![image](https://user-images.githubusercontent.com/27071950/231894427-e8bfedfd-88db-462c-990f-a0d0cc45a0ab.png)

one solution for this would be clipping the text using `contentOffsetLeftPerc` like the images

Reply:
I will look into it.